src/test/java/com/example/board/domain/post/entity/PostTest.java

```diff
@@ -0,0 +1,65 @@
+package com.example.board.domain.post.entity;
+
+import com.example.board.domain.member.entity.Member;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class PostTest {
+
+  @Test
+  @DisplayName("Post를 생성할 수 있습니다")
+  void newPost(){
+    //given
+    Member member = new Member("김환", 25, "게임");
+
+    //when
+    Post post = new Post("RBF", "RBF를 작성합니다", member);
+    String title = post.getTitle();
+    String content = post.getContent();
+
+    //then
+    Assertions.assertThat(post.getTitle())
+        .isEqualTo(title);
+    Assertions.assertThat(post.getContent())
+        .isEqualTo(content);
```

Comment:
title, content는 이미 생성한 Post에서 가져 온 것들인데 다시 검증하는 것이 의미가 있을까요?

```suggestion
    Assertions.assertThat(post.getTitle())
        .isEqualTo("RBF");
    Assertions.assertThat(post.getContent())
        .isEqualTo("RBF를 작성합니다");
```

와 같이 검증하는 것이 더 나아 보이네요

Reply:
반영해보겠습니다! 실제값은 상수로 비교하는게 더 좋다는 뜻이죠?